src/App.tsx

```diff
@@ -1,24 +1,274 @@
 /* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useEffect, useMemo, useState } from 'react';
+import classNames from 'classnames';
 
-const USER_ID = 0;
+import { Todo } from './types/Todo';
+import { FilterTypes } from './types/FilterTypes';
+import { ErrorTypes } from './types/ErrorTypes';
+
+import { getTodos, createTodo, deleteTodo } from './api/todos';
+import { RequestTodoBody } from './types/RequestTodo';
+
+const USER_ID = 10329;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState<ErrorTypes>(ErrorTypes.NoError);
+  const [isError, setIsError] = useState(false);
+  const [filterType, setFilterType] = useState<FilterTypes>(FilterTypes.All);
+  const [todoTitle, setTodoTitle] = useState('');
+  const [inputDisable, setInputDisable] = useState(false);
+  const [tempTodo, setTempTodo] = useState<RequestTodoBody | null>(null);
+  const [waitForDeletingTodoId, setWaitForDeletingTodoId] = useState<number | null>(null);
+
+  const visibleTodos = useMemo(() => {
+    const preperedTodos = [...todos];
+
+    switch (filterType) {
+      case FilterTypes.Active:
+        return preperedTodos.filter(todo => !todo.completed);
+
+      case FilterTypes.Complited:
+        return preperedTodos.filter(todo => todo.completed);
+
+      default:
+        return preperedTodos;
+    }
+  }, [todos, filterType]);
+
+  const hideNotifications = () => {
+    setTimeout(() => {
+      setIsError(false);
+    }, 3000);
+  };
+
+  const fetchTodos = async () => {
+    try {
+      const newTodos = await getTodos(USER_ID);
+
+      setTodos(newTodos);
+      setInputDisable(false);
+    } catch (error1) {
+      setError(ErrorTypes.UnableToShowTodos);
+      setIsError(true);
+      hideNotifications();
+    }
+  };
+
+  const hendlerAddTodo = async () => {
```

Comment:
Consider renaming the 'hendlerAddTodo' function to 'handleAddTodo' for consistency and better readability